frontend/src/screens/peers/Peers.tsx

```diff
@@ -18,18 +20,19 @@ import {
   TableHeader,
   TableRow,
 } from "src/components/ui/table.tsx";
-import { toast } from "src/components/ui/use-toast.ts";
+import { toast } from "src/components/ui/use-toast";
 import { useChannels } from "src/hooks/useChannels";
 import { usePeers } from "src/hooks/usePeers.ts";
 import { useSyncWallet } from "src/hooks/useSyncWallet.ts";
-import { Node } from "src/types";
+import { Node, Peer } from "src/types";
 import { request } from "src/utils/request";
 
 export default function Peers() {
   useSyncWallet();
-  const { data: peers, mutate: reloadPeers } = usePeers();
+  const { data: peers } = usePeers();
   const { data: channels } = useChannels();
   const [nodes, setNodes] = React.useState<Node[]>([]);
+  const [openPeer, setOpenPeer] = React.useState<Peer | null>(null);
```

Comment:
```suggestion
  const [openPeer, setOpenPeer] = React.useState<Peer>();
```

Reply:
If it can't be null then the dialog would be open always...

Reply:
Unless you pass a value to `useState` then the type will be `Peer | undefined` which means the default will be `undefined`. We do this much more often than using null